docs/changelogs/CHANGELOG-2.27.md

```diff
@@ -0,0 +1,241 @@
+- A regression in 2.26.0 started overriding the `floatingIPPool` fields of OpenStack clusters with the default external network. If you are using a floating IP pool that is not the default external network, you might have to update `Cluster` objects manually after upgrading KKP to set the correct floating IP pool again ([#13834](https://github.com/kubermatic/kubermatic/pull/13834))
+
+### API Changes
+
+- Add `spec.componentsOverride.prometheus` to allow overriding Prometheus replicas and tolerations ([#13893](https://github.com/kubermatic/kubermatic/pull/13893))
+- Tagged KKP releases will no longer tag the KKP images twice (with the Git tag and the Git hash), but only once with the Git tag. This is so that existing hash-based container images do not suddenly change when a Git tag is set and the release job is run. Users of tagged KKP releases are not affected by this change ([#13763](https://github.com/kubermatic/kubermatic/pull/13763))
+
+### Supported Kubernetes Versions
+
+- Remove support for Kubernetes 1.28 ([#13968](https://github.com/kubermatic/kubermatic/pull/13968))
+- Add support for Kubernetes 1.32 ([#13984](https://github.com/kubermatic/kubermatic/pull/13984))
+- Add 1.32.1/1.31.5/1.30.9/1.29.13 to the list of supported Kubernetes releases ([#14059](https://github.com/kubermatic/kubermatic/pull/14059))
+
+#### Supported Versions
+
+- 1.29.13
+- 1.30.9
+- 1.31.5
+- 1.32.1
+
+### Cloud Providers
+
+#### AWS
+
+- Update AWS cloud-controller-manager to 1.31.1 ([#13838](https://github.com/kubermatic/kubermatic/pull/13838))
+- Update AWS EBS CSI addon to 1.32.0 ([#13508](https://github.com/kubermatic/kubermatic/pull/13508))
+
+#### Azure
+
+- Add images field to DatacenterSpecAzure, allowing configuration of default OS images for Azure datacenters in Seeds ([#13924](https://github.com/kubermatic/kubermatic/pull/13924))
+
+#### DigitalOcean
+
+- Downgrade DigitalOcean CCM from 0.1.57 to 0.1.56 to work around machine-controller limitation ([#14149](https://github.com/kubermatic/kubermatic/pull/14149))
+
+#### GCP
+
+- Bump GCP CSI driver to 1.15.0 ([#13800](https://github.com/kubermatic/kubermatic/pull/13800))
+
+#### KubeVirt
+
+- Add KubeVirt DS in the charts repo to generate images for the mirrored images command ([#14064](https://github.com/kubermatic/kubermatic/pull/14064))
+- Add `VolumeProvisioner` field in the InfraStorageClasses for the KubeVirt provider in the seed, to indicate whether the storage class can be used by the CSI or the CDI ([#14111](https://github.com/kubermatic/kubermatic/pull/14111))
+- Support `CSIDriverOperator` field in the seed object to customize csi driver images in the KubeVirt user cluster ([#14147](https://github.com/kubermatic/kubermatic/pull/14147))
+- Support KubeVirt VMs LiveMigrate as an eviction strategy ([#14076](https://github.com/kubermatic/kubermatic/pull/14076))
+- Bump KubeVirt csi-driver-operator to support zone-aware topologies ([#13833](https://github.com/kubermatic/kubermatic/pull/13833))
```

Comment:
There are multiple KubeVirt CSI entries here, can be combine them to be just 1 item (we can still list the PR IDs all individually), like in

> Update KubeVirt CSI Driver Operator to (whatever is the latest PR updating it to) (#1, #7, #192, #...)